.travis.yml

```diff
@@ -0,0 +1,215 @@
+language: go
+
+go:
+  - 1.13.x
+
+os: linux
+
+notifications:
+  email:
+    on_success: never
+
+cache:
+  directories:
+    - /usr/local/kubebuilder/bin
+    - $HOME/.cache/go-build
+    - $HOME/gopath/pkg/mod
+
+stages:
+  - preparation
+  - build
+  - deploy
+  - manifest
+  - release
+
+env:
+  global:
+    - GCR=false
+    - TAG=$TRAVIS_TAG
+    - VERSION_TAG="/^v.*/"
+    - DOCKER_CLI_EXPERIMENTAL=enabled
+
+######################### DEFINING BASE STEPS #########################
+x_base_steps:
+  - &build
+    script: ./build/build.sh
+    cache:
+        directories:
+          - $HOME/.cache/go-build
+          - $HOME/gopath/pkg/mod
+
+  - &deploy
+    script: ./build/deploy.sh
+
+  - &create_manifest
+    stage: manifest
+    install: skip
+    git:
+      clone: false
+    script:
+      - docker manifest create $IMAGE:$TAG $IMAGE:$TAG-arm64 $IMAGE:$TAG-amd64
+      - docker manifest push $IMAGE:$TAG
+    workspaces:
+      use: dockerconfig
+
+jobs:
+  include:
+    - stage: preparation
+      name: Tests
+      script: ./build/test.sh
+      cache:
+        directories:
+          - /usr/local/kubebuilder/bin
+
+    - stage: preparation
+      name: Verify OLM manifest
+      language: python
+      python: "3.7"
+      script:
+        - pip3 install operator-courier==2.1.7
+        - operator-courier verify ./deploy/olm/kubernetes
+        - operator-courier verify ./deploy/olm/openshift
+
+    - stage: preparation
+      name: Set configs
+      install: skip
+      language: minimal
+      git:
+        clone: false
+      script:
+        - mkdir -p ~/.docker && chmod 0700 ~/.docker
+        - touch ~/.docker/config.json && chmod 0600 ~/.docker/config.json
+        - base64 -d >~/.docker/config.json <<<"$OAO_DOCKER_AUTH"
+      workspaces:
+        create:
+          name: dockerconfig
+          paths:
+            - $HOME/.docker
+
+    ######################### BUILD #########################
+
+    - <<: *build
+      stage: build
+      name: Build for AMD
+      arch: amd64
+      workspaces:
+        create:
+          name: ws_amd
+          paths:
+            - LICENSE
+            - build
+
+    - <<: *build
+      stage: build
+      name: Build for ARM
+      arch: arm64
+      workspaces:
+        create:
+          name: ws_arm
+          paths:
+            - LICENSE
+            - build
+
+    ######################### AMD #########################
+
+    - <<: *deploy
+      stage: deploy
+      name: Deploy snapshot master AMD
+      arch: amd64
+      if: branch = master
+      env: IMAGE=$OAO_IMAGE_DOCKER_HUB TAG="snapshot"
+      workspaces:
+        use:
+          - ws_amd
+          - dockerconfig
+
+    - <<: *deploy
+      name: Deploy snapshot quay AMD
+      arch: amd64
+      if: branch != master
```

Comment:
I wonder if `branch != master` also includes release tags?

Reply:
Added an additional check if the tag is blank